frontend/assets/sass/bulma_overrides/elements/form.sass

```diff
@@ -0,0 +1,778 @@
+    padding-left: $gi-spacer
+    padding-right: $gi-spacer
+    color: $text
+    border-color: $grey-light
+
+.input
+  padding-left: $gi-spacer
+
+  @include mobile
+    font-size: 1rem /* A11Y purposes on mobile https://stackoverflow.com/a/6394497/4737729 */
+
+  @include placeholder
+    color: rgba($input-color, 0.4) // To improve A11Y
+
+.textarea
+  padding: $gi-spacer-sm
+  line-height: 1.3
+
+  &:hover
+    border-color: $grey
+
+  &:focus
+    border-color: $primary
+    outline: $input-focus-box-shadow-size
+
+// Flat inputs like in "Contributions" page
+// TODO - review forms consistence in a future PR
+.field.gi-has-addons
+  .button
+    position: relative
+    border-color: $input-border-color
+    border-width: 1px 0 1px 0
+    border-radius: 0
+    padding: 0 $gi-spacer-sm
+
+    // Hide box-shadow between input and first button during :focus
+    &::before
+      content: ""
+      position: absolute
+      width: 6px
+      background-color: $input-background-color
+      top: 2px
+      left: -3px
+      height: calc(100% - 4px)
+
+    &:last-of-type
+      border-radius: 0 $radius $radius 0
+      border-right-width: 1px
+      padding-right: $gi-spacer
+
+    &:hover,
+    &:focus
+      color: $text !important // overide has-text-* class
+ 
+  .input
+    height: auto
+    padding-left: $gi-spacer
+    padding-right: $gi-spacer
+    border-right: 0
+    border-top-right-radius: 0
+    border-bottom-right-radius: 0
+
+    &:hover ~ .button
+      border-color: $input-hover-border-color
+
+    &:focus ~ .button
+      border-color: $primary
+      box-shadow: $input-focus-box-shadow-size $input-focus-box-shadow-color
+
+.gi-tick
+  margin-right: $gi-spacer
+
+  &-input
+    position: absolute
+    top: 1px
+    left: 1px
+    -moz-appearance: none
+    -webkit-appearance: none
+
+  &-custom
+    cursor: pointer
+    width: 0.875rem
+    height: 0.875rem
+    margin-right: $gi-spacer-xs
+    display: inline-block
+    border-radius: 50%
+    border: 1px solid $grey
+    transform: translateY(2px) // align with text
+
+    &::after
+      content: ""
+      width: 0.4rem
+      height: 0.4rem
+      background-color: $primary
+      border-radius: 50%
+      position: absolute
+      top: 50%
+      left: 50%
+      transform: translate(-50%, -50%) scale(0)
+      transform-origin: 50% 50%
+      transition: transform 150ms
+
+    .gi-tick-input:focus + &,
+    .gi-tick-input:checked + &
+      outline: none
+      border-color: $text
+
+      &::after
+        transform: translate(-50%, -50%) scale(1)
+
+.gi-
+  &fieldGroup
+    display: block
+
+    &:not(:last-child)
+      margin-bottom: $gi-spacer
+
+  &help
+    margin: -$gi-spacer-sm 0 $gi-spacer-sm
+
+    &:last-child
+      margin-bottom: 0
+
+  &is-grouped-reverse
+    flex-direction: row-reverse
+
+    &.field.is-grouped-right
+      justify-content: flex-start
+
+    &.field.is-grouped > .control:not(:last-child)
+      margin-left: 0.75rem
+      margin-right: 0
```

Comment:
You might have already explained this but I don't remember the answer.

Is this a copy/paste from bulma? If so, why is this being done?

Reply:
When there is too much changes from Bulma, it's cleaner to directly add bulma code and modify it. It's also easier to understand what's happening when you can see the full css of the component in one file.
The only reason to keep unmodified bulma file is to see that some part never been changed.

Reply:
I have two concerns with this approach:

1. It's not clear what changes you're making to bulma with this approach
2. If bulma updates itself, and diverges from your changes here, then it's possible that there could be a conflict between bulma and our changes, in which case it would be better to simply fork bulma completely and avoid having to merge back changes

Thoughts? Also would like to hear @sandrina-p's thoughts if she's around.